### DISCUSSION TITLE
InnerSource と組織論 - 次世代組織のかたちはこれだ!!

### DISCUSSION THREAD

**Discussion by yuhattor** [InnerSourceCommons/jp-roadmap#34] [Meetup]
吉田先生にぜひお話ししていただきたい...
@uikou さんとか、コメントいかがでしょう

**Comment by MototumiYoshida** [comment]
ぜひ考えまとめて、皆さんと議論したいです！よろしくお願いします。

**Comment by uikou** [comment]
@MototumiYoshida さん、ちょっと組織の話と遠いところではあるのですが、個人的に Mass さんが書かれているこの辺りはとても面白いテーマだと思っています。経営側の視点とあわせてみて、どんな思考のギャップがあるのか、また理解をし合うことで仮説の方向が変わったりしていくのかというあたりなどご意見いただいてみたいです。
https://github.com/InnerSourceCommons/jp-roadmap/discussions/28
